runtime/binding-kafka/src/main/zilla/protocol.idl

```diff
@@ -581,6 +581,37 @@ scope protocol
             }
         }
 
+        scope describe_cluster
+        {
+            struct DescribeClusterRequest // v0
+            {
+                uint8 includeAuthorizedOperations;
+            }
+
+             struct ClusterBroker
+            {
+                int32 brokerId;
+                string16 host;
+                int32 port;
+                string16 rack = null;
+            }
```

Comment:
Let's move this structure down between `DescribeClusterResponse` and `DescribeClusterResponsePart2` as is is part of the response and logically belongs between these structures.